worker/src/RTC/Consumer.cpp

```diff
@@ -131,6 +131,8 @@ namespace RTC
 		{
 			if (codec.mimeType.IsMediaCodec())
```

Comment:
No curly brackets is :disappointed: but I'm matching the current style.
Reasons not to omit curly brackets: https://embeddedgurus.com/barr-code/2014/03/apples-gotofail-ssl-security-bug-was-easily-preventable/